web/client/actions/controls.js

```diff
@@ -52,6 +55,21 @@ function setControlProperty(control, property, value, toggle) {
     };
 }
 
+/**
+ * Sets a list of properties at once
+ * @memberof actions.controls
+ * @param {string} control  control name
+ * @param {object} properties the properties to set, as key (property name), value pairs
+ * @return {object} of type `SET_CONTROL_PROPERTIES` with control and properties
+ */
+function setControlProperties(control, ...properties) {
```

Comment:
Why not directly the object? I think is more explicit that the array:
```
setControlProperties('mycontrol', 'a', 1, 'b', 2); // less intuitive
setControlProperties('mycontrol', {a:1, b: 2}) // more intuitive
```
Notice. The documentation tells something different. I didn't try yet, but documenting spread operator may work using this syntax `@param {...object} myParam` (JSDoc 3.4.3). Please try (or refactor as suggested)

Reply:
Because using ...properties is more functional friendly (partial application) and in fact my first usage of it took advantage of partial application. Eventually we can also support both syntaxes

Reply:
Now I see. That's cool :+1: 
I should not do both to avoid confusion. 
Maybe here we are creating an ad-hoc API to solve a specific problem (binding connect with partial application). 

So you could do 2 different things: 
1. Change the documentation to specify better how to use the action
2. Change the `.bind` with something like ( properties ) => setControlProperties('myControl', { ...properties